### DISCUSSION TITLE
Static content

### DISCUSSION THREAD

**Discussion by eftomi** [Budibase/budibase#8015] [Help] [answered]
Hi, congratulations to Budibase team for v2.0!

Is it possible to serve static content with Budibase? For instance help pages, as html and css documents, together with their resources like gifs? Just to avoid setting up another content server.

Thanks.

**Comment by melohagan** [comment, marked as answer]
Hey @eftomi 

Late reply, but you could try using an [Embed](https://docs.budibase.com/docs/embed) component, which does support binding values as well.
